app/views/layout.erb

```diff
@@ -1,18 +1,26 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <script src="https://cdn.jsdelivr.net/jquery/1/jquery.min.js"></script>
+    <script src="<%=url('interface.js')%>"></script>
     <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
     <link href='https://fonts.googleapis.com/css?family=Cardo' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" type="text/css" href="/reset.css"/>
     <link rel="stylesheet" type="text/css" href="/main.css"/>
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/bootstrap/latest/css/bootstrap.css"/>
```

Comment:
require order: bootstrap.css before main.css to avoid overwriting our style (thx anne)